doc/source/Platforms.rst

```diff
@@ -282,26 +308,6 @@ The following is required for building new spack environments and for using spac
    module load miniconda/3.9.12
    module load ecflow/5.8.4
 
-For ``spack-stack-1.0.1`` with Intel, load the following modules after loading miniconda and ecflow:
-
-   ulimit -s unlimited
```

Comment:
Why do we list `ulimit -s unlimited` in some recipes and not in others. Don't we always want to use "unlimited" stack size?

Reply:
You are right, I removed it everywhere except where it's listed as recommendation for generic Linux/macOS.

Reply:
Sounds great. thanks!